server/container_create.go

```diff
@@ -384,6 +384,27 @@ func ensureSaneLogPath(logPath string) error {
 	return nil
 }
 
+// addSecretsBindMounts mounts user defined secrets to the container
+func addSecretsBindMounts(mountLabel, ctrRunDir, configDefaultMountsPath string, specgen generate.Generator) error {
+	mountPaths := []string{libkpod.OverrideMountsFile, libkpod.DefaultMountsFile}
+	// configDefaultMountsPath is used to override the mount file path for testing purposes only when set in the runtime config
```

Comment:
Why do we need this override? We can just add a flag to make this uniform with how we modify other defaults in cri-o for testing.

Reply:
Was following the systemd standards as mentioned by @rhatdan in one of the review comments above:

> The standard I follow was established by systemd.
We put content in /usr/share/APP/mounts.d/foo or /usr/lib/APP/mounts.d/foo and it gets processed. If an admin comes in and adds /etc/APP/mounts.d/bar, then foo and bar get processed. If the admin adds /etc/APP/mounts.d/foo, then Only the Admins foo and bar get processed. IE He has overridden the systems foo.
We will ship a default package that mounts the RHEL secrets, this should be under /usr. We should have a goal of nothing required to be installed in /etc. /usr == Distributions stuff. /etc/==admin stuff.
There is a long stated goal to make a system reset be done by executing rm -rf /etc /var; reboot
If our services can not survive that, we should make them.